reflection_refraction.py

```diff
@@ -89,19 +89,22 @@ def matplotlib_graph(p1, p2, p3, n1, n2):
     # TODO: Print angles
 
     # With Vectors
-    # color codes: #ff0a0a; #ff4141
+    
+    # Color codes
+    color_i = '#ff0a0a'
+    color_rr = '#ff4141'
```

Comment:
Constants in python are usually written in all capital letters with underscores separating words. See [the conventions](https://www.python.org/dev/peps/pep-0008/#constants).

Also, if you want the constants to be part of the same module, they are used in, usually they are written at the beginning of the module.

Reply:
Ok

Reply:
Thank you for changing it.